views/website/libraries/19-Swift.json

```diff
@@ -160,6 +160,41 @@
       "gitHubRepoPath": "airsidemobile/JOSESwift",
       "repoUrl": "https://github.com/airsidemobile/JOSESwift",
       "installCommandHtml": "pod 'JOSESwift'"
+    },
+    {
+      "minimumVersion": null,
+      "support": {
+        "sign": true,
+        "verify": true,
+        "iss": true,
+        "sub": true,
+        "aud": true,
+        "exp": true,
+        "nbf": true,
+        "iat": true,
+        "jti": true,
+        "typ": true,
+        "hs256": true,
+        "hs384": true,
+        "hs512": true,
+        "rs256": true,
+        "rs384": true,
+        "rs512": true,
+        "es256": true,
+        "es384": true,
+        "es512": true,
+        "es256k": true,
+        "ps256": true,
+        "ps384": true,
+        "ps512": true,
+        "eddsa": true
+      },
+      "authorUrl": "https://github.com/beatt83",
+      "authorName": "Gonçalo Frade",
+      "gitHubRepoPath": "beatt83/jose-swift",
+      "repoUrl": "https://github.com/beatt83/jose-swift",
+      "installCommandHtml":
+        ".package(url: \"https://github.com/your-username/jose-swift.git\", .upToNextMinor(from: \"1.1.0\"))"
```

Comment:
Could you please verify if the URL [https://github.com/your-username/jose-swift](https://github.com/your-username/jose-swift) on this command for installing the library is correct? I'm getting a 404 for it.

Reply:
Hi @byron-okta . Thx for the review :). Yes it was wrong it's now fixed I took the time to update the version as well for the most recent.